### ISSUE TITLE
Add indicators for required fields

### ISSUE THREAD

**Issue by Studsministern** [Studsministern/odin-sign-up-form#5]
Add "*" either next to the label for each input field, or as a warning text after a failed validation. This indicator should also support screenreaders.
